### DISCUSSION TITLE
Trouble connecting to server

### DISCUSSION THREAD

**Discussion by VaguelyPro** [ORelio/Minecraft-Console-Client#2487] [General]
I get this msg every time I try to connect. https://prnt.sc/7m9j_z6ZHhd7

**Comment by milutinke** [comment]
> I get this msg every time I try to connect. https://prnt.sc/7m9j_z6ZHhd7

Stop the client.
Delete `SessionCache.ini`, delete `KeysCache.ini`
Open up the `MinecraftClient.ini`, set the `Method` field to `"browser"`
Save the file.
Start the client and click on the provided link, log in and copy the code you got, paste in the code in to the client, then press enter.
Then try to type in the server ip.

**Reply by VaguelyPro** [reply]
I tried that with no luck. Does it matter what type of server it is? The server is running on paper

**Reply by milutinke** [reply]
> I tried that with no luck. Does it matter what type of server it is? The server is running on paper
> 

Are you maybe trying to log in to the server that that is paid account only with an offline (non paid) account?
Also, it could be that the server software itself has been modified and is not working as expected.
Try joining other servers.
Also, make sure that you don't have any special non allowed characters in your username if you are using a non paid (offline) account.

**Reply by VaguelyPro** [reply]
It's a normal paid minecraft account. It lets me join other servers. Do you know what server settings would be preventing this?

**Reply by VaguelyPro** [reply]
I figured it out. I just needed to allow proxies on the server
